source/use-nunjucks/index.html.md.erb

```diff
@@ -0,0 +1,74 @@
+---
+title: Use Nunjucks
+weight: 45
+---
+
+# Use Nunjucks
+
+You can use our Nunjucks code to generate the HTML for pages and components.
+
+You can also change the HTML that's generated by passing options into the Nunjucks code.
+
+If you use Nunjucks, your project will be easier to update when a new version of GOV.UK Frontend is released.
+
+To use Nunjucks:
+
+- you must [install GOV.UK Frontend with Node.js package manager (npm)](/installing-with-npm/)
+- your application and any frameworks that support it must use Node.js
+
+You do not need to follow this guidance to [use Nunjucks macros in the GOV.UK Prototype Kit](https://design-system.service.gov.uk/get-started/prototyping/#using-nunjucks-macros).
+
+## Set up Nunjucks and use the page template
+
+1. Make sure you installed Nunjucks when you [installed GOV.UK Frontend with npm](/installing-with-npm).
+
+2. Import the Nunjucks package at the top of your configuration file, for example:
+
+    ```javascript
+    const nunjucks = require('nunjucks')
+    ```
+
+3. Tell Nunjucks where to find the GOV.UK Frontend template and components by adding the following to your configuration file:
+
+    ```javascript
+    nunjucks.configure([
+      "node_modules/govuk-frontend/"
+    ], {
+      autoescape: true
+    })
+    ```
+
+    `autoescape: true` makes the output render as text by default, to help prevent cross-site scripting (XSS) attacks.
+
+4. Use the GOV.UK Frontend template by adding the following at the top of your view file:
```

Comment:
There's a step missing here which is where the configured nunjucks environment is actually hooked in to the view layer of whatever framework they're using.

Unfortunately, what that step looks like is going to depend a lot on what framework is being used.